### PR TITLE
Release version 1.3.0

### PR DESCRIPTION
**Pull request by srinivas-rao5**
**Ansible modules for Dell EMC PowerStore system release version 1.3.0 includes the following:**

- Added new modules: cluster, network, local user, job, and roles.
- Enhancements in gatherfacts module.
- Incorporated dual licensing.
- Minor bug fixes.

### CODE REVIEW

**Review by shenda1 — APPROVED** [review]

**Review by dattaarindam — APPROVED** [review]

**Review by dattaarindam — APPROVED** [review]

**Review by dattaarindam — APPROVED** [review]

File: ChangeLog.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # ansible-powerstore Change Log
+## Version 1.3.0 - released on 28/09/2021
+- Added dual licensing.
+- Added CRUD operations for local user.
+- Network module is added to get the details and modify attributes of network.
+- Cluster module is added to get the details and modify attributes of cluster.
+- Job module is added to get the details of given job ID.
+- Role module is added to get the details of given role name or role ID.
+- Gather facts module is enhanced to list users, roles, networks and appliances.
+- Added dual licensing.
```
Comment (suman-dell) [inline ChangeLog.md:10]: Remove it - "dual licensing" mentioned twice.

File: galaxy.yml [hunk 2]
```diff
@@ -63,5 +65,4 @@ issues: https://www.dell.com/community/Automation/bd-p/Automation
 # artifact. A pattern is matched from the relative path of the file or directory of the collection directory. This
 # uses 'fnmatch' to match the files or directories. Some directories and files like 'galaxy.yml', '*.pyc', '*.retry',
 # and '.git' are always filtered
-build_ignore: []
-
+build_ignore: []
```
Comment (suman-dell) [inline galaxy.yml:68]: newline needed at the end of file - we usually do it with dos2unix command.

**Review by suman-dell — COMMENTED** [review]
1 minor change needed

File: ChangeLog.md (hunk 1 above)
Comment (dattaarindam) [reply ChangeLog.md]: Addressed. Thanks.

**Pull request closed without merging**